<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSetWidget</class>
 <widget class="QWidget" name="LevelSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>215</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose a level set to play:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_lwLevelSets"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KNSWidgets::Button" name="m_pbNewStuff">
       <property name="text">
        <string>Download New Levelsâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KNSWidgets::Button</class>
   <extends>QPushButton</extends>
   <header location="global">KNSWidgets/Button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
